app/views/people/index.html.erb

```diff
@@ -1,49 +1,25 @@
 <!--SEARCHBROWSE Descriptive page name, messages and instructions-->
 
 <div class="pseud">
```

Comment:
Could you remove this surrounding div? There's no godly reason for it to be here -- I wouldn't be surprised if it were left over from layout 1.0